cime_config/testdefs/testlist_clm.xml

```diff
@@ -672,32 +648,6 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="ERP_Ld5" grid="f19_g17" compset="I1850Clm50Bgc" testmods="clm/default">
-    <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:20:00</option>
-    </options>
-  </test>
-  <test name="ERP_Ld5" grid="f19_g17" compset="I2000Clm50BgcCru" testmods="clm/default">
-    <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:20:00</option>
-    </options>
-  </test>
-  <test name="ERP_Ld5" grid="f19_g17" compset="I2000Clm50SpRtmFl" testmods="clm/default">
```

Comment:
I think removing the top two tests should be fine. I think the ctsm_sci essentially duplicates these. The Flooding test (I2000Clm50SpRtmFll)  think should remain at a lower resolution, or be moved to the ctsm_sci test list. I think Flooding was only originally tested at f09/f19 resolution, and possibly only configured for that. It looks like this should now work at other resolutions though as far as I can tell. The flooding test is important for testing with the coupler since it adds additional fields being passed through the mediator.

- [ ] Add back the Flooding test at a lower resolution.